### ISSUE TITLE
Smart! for Signs.

### ISSUE THREAD

**Issue by snowboundmage2** [SmartFoundations/SmartIssueTracker#87]
Humbly requesting that Smart add support for sign placement.
Larger factories usually have repetitive structural supports, which can be prettied up with signs. Signs can be used as lighting, or as decoration, to prevent blandness.
Smart! signs can also be used along machines, if people want to have displays over constructers saying what item is being built, how many, etc.

(This might be painful to do, there is plenty of signs sizes.)

**Describe alternatives you've considered**
Here is an alternative for those looking to place signs along containers: https://ficsit.app/mod/zopCPnSnG4k3T

**Additional context**
At least 2 people in the discord asked for it.

**Comment by ShadedPL** [comment]
I totally agree, Smart for signs, especially when able to copy the sign itself would be such a great time-saver

**Comment by majormer** [comment]
This issue has been migrated to the new repository.

**New location:** https://github.com/majormer/SmartFoundations/issues/86

Please follow up there.
